### DISCUSSION TITLE
Since updating my mouse pointer doesn't pick which monitor to control

### DISCUSSION THREAD

**Discussion by tenapercy-sh** [MonitorControl/MonitorControl#1650] [General]
I updated to sequoia and then updated to the latest version of MonitorControl and now my brightness buttons don't control the correct monitor. I have a MBP and a Samsung external display. I have the setting "Depends on mouse pointer control" but it always chooses the built in display.

**Comment by waydabber** [comment]
You'll probably need to reconfigure Accessibility permissions under System Settings/Privacy.
